src/components/UI/Dropdown/index.js

```diff
@@ -1,19 +1,73 @@
-import React from "react";
-import i18next from "i18next";
-// Styles
-import { Container } from "./dropdown.styled";
+import React, { useState } from "react";
+import styled from "styled-components";
+import {IoIosArrowDown} from 'react-icons/io';
 
-export function Dropdown() {
-  const handleLanguageChange = (lang = "en") => {
-    i18next.changeLanguage("sr");
-  };
+
+export function Dropdown(props) {
+  const [isOpen, setIsOpen] = useState(false);
+  const toggling = () => setIsOpen(!isOpen);
+  const [selectedOption, setSelectedOption] = useState(props?.list[0]);
+
+    const onOptionClicked = value => {
+    setSelectedOption(value);
+    setIsOpen(false);
+    props.onSelect(value);
+};
```

Comment:
For event handlers, currently the names are `handleLanguageChange`, `toggling` & `onOptionClicked`.
All three have totally different approach to naming. Can we have consistent naming for all event handlers? We should pick one approach and stick to it.

Reply:
Of course. I understand everthing. I will change. :)